r/vignettes/informal_introduction.Rmd

```diff
@@ -0,0 +1,297 @@
+---
+title: Getting started with Apache Arrow and R  
+description: >
+  An informal introduction of the functioning of Apache Arrow for R users
+output: rmarkdown::html_vignette
+---
+
+
+## What is this vignette and is it for you?
+
+This vignette provides an overview of how Arrow works in a plain, non-technical language. It aims at giving some simple and useful intuitions, before diving deeper in the documentation. It is specifically intended for newcomers with a limited background in computer science and hence avoids most technical terms. This vignette assumes that you have some experience with R, that you are familiar with the `dplyr` syntax and that you know how to use a Parquet file.
+
+Some technical points are deliberately simplified to keep things simple, so in case you find an apparent contradiction between this vignette and the rest of the documentation, please trust the documentation rather than this vignette.
+
+## Introducing Apache Arrow
+
+This section introduces the Apache Arrow project.
+
+### What is Apache Arrow?
+
+[Apache `Arrow`](https://arrow.apache.org/) is an *open-source* project that offers two things: 
+
+- Apache Arrow defines a standardized way to organize data in memory (called _Apache Arrow Columnar Format_). You do not need to know much about this _Columnar Format_ to use Arrow with R, except that it is very efficient (processing is fast) and interoperable (meaning for instance that both R and Python can access the same data, without converting the data from one format to another).
+- Apache Arrow offers a `C++` implementation of this _Columnar Format_: the `C++` library called `libarrow`.
+
+What about the Arrow R package then? This package simply makes it possible to use the `libarrow` library within `R`. Keep in mind that there are other similar interfaces for using `libarrow` with other programming languages: in Python, Java, Javascript, Julia, and so on. But __no matter what programming language you choose for using Arrow, remember that under the hood you are using exactly the same tool: the C++ libarrow library__.
```

Comment:
Of these, only R and Python are based on libarrow, while Java, JavaScript, and Julia are all independent implementations that are completely different from C++.
Please check <https://arrow.apache.org/docs/status.html>.

Reply:
thanks @eitsupi. I reformulated this part, tell me if the new wording is alright.

Reply:
I think "Keep in mind that there are other similar interfaces for using `libarrow` with other programming languages" is not correct.

Reply:
I reformulated again (thanks for the link to the documentation, I now understand better the various Arrow implementations). Tell me if this is OK for you.